core/themes/basis/css/component/node.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Styling for node content.
+ */
+
+.node {
+  position: relative;
+}
+
+/* Display "Draft" watermark on unpublished content. */
+/* See basis_preprocess_node(). */
+.node[data-status]:before {
+  content: attr(data-status);
+  position: absolute;
+  z-index: 5;
+  width: 100%;
+  top: 20%;
+  pointer-events: none;
+  text-align: center;
+  text-transform: uppercase;
+  font-size: 10em;
+  font-weight: bold;
+  opacity: 0.5;
+  color: #dee2ea;
```

Comment:
This should be darker methinks, `#ccc` has more contrast and you can still see through it.